Jenkinsfile

```diff
@@ -47,7 +47,6 @@ node {
 
     for (adapter in DB_ADAPTERS) {
       stage("Set up the DB") {
-        sh("RAILS_ENV=test SIGNONOTRON2_DB_ADAPTER=${adapter} bin/rails db:environment:set")
```

Comment:
FYI you shouldn't reintroduce this. It will error if db doesn't exist and can be better skipped with an env var: https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy#L154

Reply:
:+1: I'll remove that change in the updated PR